app/models/version.rb

```diff
@@ -315,7 +315,11 @@ def authors_array
   end
 
   def sha256_hex
-    sha256.unpack("m0").first.unpack("H*").first if sha256
+    Version._sha256_hex(sha256) if sha256
+  end
+
+  def self._sha256_hex(raw)
```

Comment:
To differentiate it from [`Versions#sha256_hex`](https://github.com/rubygems/rubygems.org/pull/1305/files#diff-3c35bc805bbb122cee7af2e3884a4d66R317)